sunpy/map/tests/test_plotting.py

```diff
@@ -79,28 +80,58 @@ def test_peek_aia171(aia171_test_map):
 
 @figure_test
 def test_peek_grid_aia171(aia171_test_map):
-    aia171_test_map.peek(draw_grid=True)
+    with warnings.catch_warnings():
+        warnings.filterwarnings("ignore", message=(
+            'The conversion of these 2D helioprojective coordinates to '
+            '3D is all NaNs because off-disk coordinates need an additional assumption to be mapped to 3D. '
+            'Consider using the context manager `Helioprojective.assume_spherical_screen()'
+        ), category=SunpyUserWarning)
```

Comment:
Locally if I use pytest.raises, the test error. 

I don't know why so I did this.